src/test/ui/expr/malformed_closure/ruby_style_closure.stderr

```diff
@@ -1,29 +1,9 @@
 error[E0425]: cannot find value `x` in this scope
-  --> $DIR/ruby_style_closure.rs:13:14
+  --> $DIR/ruby_style_closure.rs:12:14
    |
 LL |         Some(x * 2)
    |              ^ not found in this scope
 
-error[E0277]: expected a `FnOnce<({integer},)>` closure, found `Option<_>`
```

Comment:
This is kind of a shame, but this error gets suppressed because we end up with `Option<[type error]>`, which we then don't emit a selection error for.

Reply:
That's fine, particularly because of how malformed the code is here.